app/views/practices/form/_practice_last_updated.html.erb

```diff
@@ -0,0 +1,8 @@
+<% last_updated = PracticeEditorSession.practice_last_updated(@practice.id) %>
+<% if !last_updated.blank? %>
```

Comment:
I think we can change this to `unless last_updated.blank?`.

Reply:
Got it.